news/2018-11-04-project-loved-week-of-november-4th.md

```diff
@@ -0,0 +1,208 @@
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/self-producer.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Chihara Minori - SELF PRODUCER](https://osu.ppy.sh/beatmapsets/419728#osu) by [Startrick](https://osu.ppy.sh/users/2782104)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+Known to some as the opening for the anime "OniAi", the song "SELF PRODUCER" has a beautifully crafted osu! map made by Startrick! At a length of five minutes, this energetic and upbeat song is accompanied by an artistically pleasing alternating flow map. If you're a player who prides their flow aim and consistency, why not use this map to prove your skills? Check it out and drop a vote on the map's Loved poll!
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/the-government-knows.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[KNOWER - The Government Knows](https://osu.ppy.sh/beatmapsets/721707#osu) by [Woey](https://osu.ppy.sh/users/3792472), [MomoHD](https://osu.ppy.sh/users/2927048), [Ambrew](https://osu.ppy.sh/users/7803396) and [Hectique](https://osu.ppy.sh/users/5298911)  
+*written by [SoraRoxas01](https://osu.ppy.sh/users/1986262)*
+
+This is a great map for people that really want to test their skill with technical maps. "The Government Knows" is a map full of challenging patterns along with both fast and slow sliders which will test how well you can control your aim.  This song is unique in its own way with the constant dance beat in the background and humorous lyrics. You may have recognised Woey previously from his map "Soulless 4" that was a candidate for Project Loved before, along with guest difficulties from Ambrew, MomoHD and Hectique. With the map hitting over 120 favorites, this is a map to definitely give a go if you're up for a challenge!
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/concept-of-reality.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[lapix feat. Renko - Concept Of Reality](https://osu.ppy.sh/beatmapsets/559140#osu) by [ProfessionalBox](https://osu.ppy.sh/users/3250792) et al.  
+*written by [SoraRoxas01](https://osu.ppy.sh/users/1986262)*
+
+One look into this map and you'll know straight away that this is by the unique mapper ProfessionalBox. Not to mention that the song is an absolute banger! But for real though, I'm sure you didn't expect anything less from lapix! Along with ProfessionalBox, there are guest difficulties mapped by other well known mappers like NeilPerry, Sing, Low Combo FC, Haruto and Ameth Rianno. With the eccentric slider shapes and changes in slider velocity during the chorus to go with interesting fillers leading to the build up, this is a map that keeps you toes and is something to have a go at if you're into the more technical and nifty maps!
```

Comment:
```suggestion
One look into this map and you'll know straight away that this is by the unique mapper ProfessionalBox. Not to mention that the song is an absolute banger! But for real though, I'm sure you didn't expect anything less from lapix! Along with ProfessionalBox, there are guest difficulties mapped by other well known mappers like [NeilPerry](https://osu.ppy.sh/users/841391), [Sing](https://osu.ppy.sh/users/3795679), [Low Combo FC](https://osu.ppy.sh/users/3338724), [Haruto](https://osu.ppy.sh/users/3772301) and [Ameth Rianno](https://osu.ppy.sh/users/5219516). With the eccentric slider shapes and changes in slider velocity during the chorus to go with interesting fillers leading to the build up, this is a map that keeps you toes and is something to have a go at if you're into the more technical and nifty maps!
```

I think that the et al mappers should be linked since they're in the description, but it seems weird to do it here.